src/main/java/servlets/VendorServlet.java

```diff
@@ -14,20 +14,31 @@
 
 package servlets;
 
+import data.FileParser;
+
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 @WebServlet("/VendorServlet")
 public class VendorServlet extends HttpServlet {
   @Override
+  /**
+   * Prints a hashmap to the response object
+   * STRING (Vendor ID) : ARRAY OF STRINGS (Account IDs)
+   * e.g., {"vend_1":["account1","account2","account3","account4"],
```

Comment:
Why are we doing this? I though this was just returning a list of vendor IDs.

Reply:
The frontend is handling selecting a vendor and account with drop down menus. When the user selects a specific vendor, the next drop down is populated from the array of account IDs